app/views/posts/new.html.erb

```diff
@@ -0,0 +1,21 @@
+<section>
+  <h2>New Post</h2>
+  <% if @post.errors.any? %>
+    <div>
+      <% @post.errors.full_messages.each do |message| %>
+      <p><%= message %></p>
+      <% end %>
+    </div>
+  <% end %>
+  <%= form_with model: @post, url: user_posts_path(@user) do |form| %>
+    <div>
+      <%= form.label :title, class: "form-label" %>
+      <%= form.text_field :title, required: true, autofocus: true, class: "form-input" %>
+    </div>
+    <div>
+      <%= form.label :text, class: "form-label" %>
+      <%= form.text_area :text, required: true, autofocus: true, class: "form-input" %>
+    </div>
+    <%= form.submit "submit", class: "form-button" %>
+  <% end %>
+</section>
```

Comment:
- [ ] I see that you created a view for your Post form, however, you didn't create the respective actions in the PostsController. In consequence, I'm getting this error when I try to access this view:

![image](https://github.com/DJ-MrJay/My-Blog-App/assets/77069011/5653f40d-76ef-44a1-8c65-fdf4b6227cf8)

Please consider fixing this. 🤓 🔧